sphinx/config.py

```diff
@@ -123,6 +123,7 @@ class Config(object):
         primary_domain = ('py', 'env', [NoneType]),
         needs_sphinx = (None, None, string_classes),
         needs_extensions = ({}, None),
+        manpages_url_pattern = (None, 'html'),
```

Comment:
I think this should be effect not to only html, but also all builders (in future). For example, it would be nice if LaTeX builder also support the hyperlinks. \^So last argument should be `'env'`. It means the update of settings effects to all builders.

Reply:
ah. okay.